compiler/circle-mpqsolver/src/bisection/BisectionSolver.h

```diff
@@ -64,6 +64,13 @@ class BisectionSolver final : public MPQSolver
    */
   void algorithm(Algorithm algorithm);
 
+  /**
+   * @brief set path of visq_file to be used in 'auto' mode
+   * for handling which way of spliting the neural network
+   * will be the best
```

Comment:
`@brief` would be better in one line for a brief explanation.
and use `@detail` for long one.
```
@brief   set visq_file path to be used in 'auto' mode
@details this is used to handle which way (8 or 16bit) of
         splitting the neural network will be the best for accuracy.
```

Q) is like like just calling `setVisqPath` will set to auto mode?

Reply:
> is like like just calling `setVisqPath` will set to auto mode?

No. `setVisqPath` just assumes that `auto` will be used, but to use it `algorithm(Algorithm::Auto)` should be called explicitly.

Reply:
> `@brief` would be better in one line for a brief explanation.
> and use `@details` for long one.

done